Annotato/Annotato/ViewModels/Document/DocumentViewModel.swift

```diff
@@ -4,9 +4,11 @@ import Foundation
 class DocumentViewModel: ObservableObject {
     private(set) var annotations: [AnnotationViewModel]
     @Published private(set) var annotationToAdd: AnnotationViewModel?
+    private(set) var pdfDocument: DocumentPdfViewModel
```

Comment:
@zognin Note to self: To rearrange attributes in next pr, not doing in this pr to reduce difficulties of merge conflicts.